server/db/models/Product.js

```diff
@@ -0,0 +1,49 @@
+const Sequelize = require('sequelize');
+const db = require('../db');
+
+const Product = db.define('product', {
+  title: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  author: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  description: {
+    type: Sequelize.TEXT,
+  },
+  summary: {
+    type: Sequelize.TEXT,
+    allowNull: false,
+  },
+  format: {
+    type: Sequelize.ENUM('hardcover', 'paperback', 'ebook'),
+    defaultValue: 'hardcover',
+    allowNull: false,
+  },
+  ISBN13: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  imageUrl: {
+    type: Sequelize.STRING,
+    //defaultValue: find default photo for public folder
+  },
+  price: {
+    type: Sequelize.DECIMAL,
```

Comment:
Is there a currency option?  Would save us some work on the front end.  If not, we can just format the decimal as a dollar amount in the front-end.

Reply:
I looked through as well and it looked like our best bet was to just add the $ on the front end.